src/test/java/org/springframework/data/aerospike/AsyncUtils.java

```diff
@@ -0,0 +1,20 @@
+package org.springframework.data.aerospike;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.*;
+
+public class AsyncUtils {
+
+    public static void executeConcurrently(int numThreads, Callable<Void> task) throws Exception {
+        ExecutorService pool = Executors.newFixedThreadPool(numThreads);
```

Comment:
who will shutdown thread pool? e.g. in case of exception your test might block jvm to exit